src/main/java/com/ays/backend/user/model/ERole.java

```diff
@@ -0,0 +1,7 @@
+package com.ays.backend.user.model;
+
+public enum ERole {
+    ROLE_VOLUNTARY,
```

Comment:
```suggestion
    ROLE_VOLUNTEER
```
voluntary yapılan işi nitelendirirken volunteer yapan insanı nitelendiriyor aynı worker gibi, daha uygun gibi geldi

Reply:
I just changed it as ROLE_VOLUNTEER instead of ROLE_VOLUNTARY